source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -1,7 +1,7 @@
 runOn:
     -
         minServerVersion: "4.3.1"
-        topology: ["replicaset, sharded"]
+        topology: ["replicaset", "sharded"]
```

Comment:
Now that these test run, the driver ignores all abortTransaction command errors, so it wouldn't get a server error to have labels on line 96: https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#drivers-ignore-all-aborttransaction-errors

Reply:
@iwysiu I'm not sure I follow is the test redundant or worth keeping to ensure `RetryableWriteError` is omitted?

Or is it that `abortTransaction only retries once with RetryableWriteError from server` is actually wrong as it should be ignored?

Reply:
sorry, I explained it badly. `abortTransaction does not retry without RetryableWriteError label` shouldn't have a result.errorLabelsOmit field because the driver shouldn't return the abortTransaction error to the user.

Reply:
Got it - thanks :) I've updated that test also.